stylesheets/print.scss

```diff
@@ -0,0 +1,56 @@
+@page {
+  margin-top: 1.5cm;
+  margin-right: 1cm;
+  margin-left: 1cm;
+  margin-bottom: 2cm;
+}
+
+body {
+  font: 12pt Georgia, "Times New Roman", Times, serif;
+}
+
+nav, footer, div.sidebar {
+  display: none;
+}
+
+// print background image and color
+header h1, p > code, li > code {
+  -webkit-print-color-adjust: exact;
+}
+
+h1, h2, h3, h4, h5 {
+  page-break-after: avoid;
+}
+
+blockquote {
+  page-break-inside: avoid;
+}
+
+pre code {
+  page-break-inside: avoid;
+}
+
+p {
+  orphans: 2;
+  widows: 2;
+}
+
+// hanging punctation
+ul {
+  list-style: disc outside none;
+  padding: 0;
+}
+
+// link handling
```

Comment:
I did not yet make up my mind if the following rules are an improvement or not.

The first two rules print the URL target after the anchor tag. Most pages only have few links to other internal or external pages (except the implementation page). Therefore I'm not sure if this behavior improves the printed pages or not.

The specification has a lot of internal links, which have no benefit on printed paper as tapping a piece of paper does almost nothing :sunglasses: The last rule hides all fragment links but now I have no indication about cross-reference anymore.